.ci/Dockerfile.opendistro

```diff
@@ -0,0 +1,7 @@
+ARG OPENDISTRO_VERSION
```

Comment:
Why are we setting up the CI with "opendistro" instead of "opensearch" when we already have the "namespace changes" in the latest commit?